distribution/helm/vector-agent/values.yaml

```diff
@@ -30,16 +30,26 @@ fullnameOverride: ""
 
 updateStrategy: RollingUpdate
 
-# Deploy service to use Topology-aware traffic routing with topology keys.
+# Deploy a service resource to use Topology-aware traffic routing with topology keys.
 service:
+  # Whether to create service resource or not.
   enabled: false
-  # type: ClusterIP
-  # port: 80
-  # topologyKeys:
-  #   - "kubernetes.io/hostname"
-  #   - "topology.kubernetes.io/zone"
-  #   - "topology.kubernetes.io/region"
-  #   - "*"
+  type: ClusterIP
+  topologyKeys:
+    - "kubernetes.io/hostname"
+    - "topology.kubernetes.io/zone"
+    - "topology.kubernetes.io/region"
+    - "*"
+  ports:
+    - name: api
+      port: 80
+      protocol: TCP
+      targetPort: 80
```

Comment:
🤔 I'm curious about moving this to a higher port, there's no need for us to add privileges if we don't need to. I didn't think of this on the PR that added this, perhaps we should default this to 8686 - the [default port for the API](https://vector.dev/docs/reference/api/#address).

Reply:
Yeah, totally agree. Will fix this.